### PR TITLE
Delete and redo file sets

### PR DESCRIPTION
**Pull request by orangewolf**

### CODE REVIEW

**Review by ShanaLMoore — DISMISSED** [review]

File: app/jobs/bulkrax/delete_file_set_job.rb [hunk 1]
```diff
@@ -2,4 +2,22 @@
 
 module Bulkrax
   class DeleteFileSetJob < DeleteJob; end
+
+  def perform(entry, importer_run)
```
Comment (ShanaLMoore) [inline app/jobs/bulkrax/delete_file_set_job.rb:6]: Do you need to pass the importer_run? It doesn't seem like it's getting used here.

File: app/jobs/bulkrax/delete_file_set_job.rb [hunk 2]
```diff
@@ -2,4 +2,22 @@
 
 module Bulkrax
   class DeleteFileSetJob < DeleteJob; end
```
Comment (ShanaLMoore) [inline app/jobs/bulkrax/delete_file_set_job.rb:4]: Also, ignore me if I'm reading this wrong, but is this an extra end?

File: app/jobs/bulkrax/delete_file_set_job.rb (hunk 1 above)
Comment (orangewolf) [reply app/jobs/bulkrax/delete_file_set_job.rb:6]: its used in the parent job (so the call to super gets it)

File: app/jobs/bulkrax/delete_file_set_job.rb (hunk 2 above)
Comment (orangewolf) [reply app/jobs/bulkrax/delete_file_set_job.rb]: grr - I really hate the `; end` syntax. it takes an important thing and hides it for no real gain but to save a linefeed. fixed

File: app/jobs/bulkrax/delete_file_set_job.rb (hunk 1 above)
Comment (orangewolf) [reply app/jobs/bulkrax/delete_file_set_job.rb]: this was a really good catch Shana. its legal to define the method on the module so its syntaxitcally correct but would not have ever been called the way I had it

File: app/jobs/bulkrax/delete_file_set_job.rb (hunk 2 above)
Comment (orangewolf) [reply app/jobs/bulkrax/delete_file_set_job.rb]: this was a really good catch Shana. its legal to define the method on the module so its syntaxitcally correct but would not have ever been called the way I had it

**Review by ShanaLMoore — APPROVED** [review]

**Pull request merged**